### PR TITLE
Hotfix: 'resources' searched in header now takes user to 'data' page

### PR DESCRIPTION
**Pull request by Tehsurfer**
# Description

Hotfix to take users to /data from the header search.

Previously users were taken to the 'Tools and Resources' page.

Wrike ticket defining this can be found here:
https://www.wrike.com/open.htm?id=869259065

## Type of change

Delete those that don't apply.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Tested locally. 

Seeing as it is quite a small change I did not create an instance for testing. The lines changed here should hopefully be self explanatory

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have utilized components from the Design System Library where applicable
- [ ] I have added unit tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by hsorby — COMMENTED** [review]
I would rather see this as a single if clause that overrides the default values, rather than a series of inline ifs.

**Comment by Tehsurfer** [comment]
> I would rather see this as a single if clause that overrides the default values, rather than a series of inline ifs.
@hsorby 
Is this what you were thinking?:

```
    executeSearch: function(term) {
      const option = this.searchSelectOptions.find(
        o => o.value === this.searchSelect
      )
      let type = option.value
      let routeParams = {
        name: type,
        query: {
          type,
          seach: term
        }
      }

      if (option.value === 'data'){
        let type = 'dataset'
        routeParams = {
          name: 'dataset',
          query: {
            type,
            q: term
          }
        }
      } else if (option.value === 'resources') {
        let type = 'sparcPartners'
        routeParams = {
          name: 'data',
          query: {
            type,
            q: term
          }
        }
      }

      this.$router.push(routeParams)
     ```

**Comment by alan-wu** [comment]
I think we need to clarify the behaviour before merging this in. The new behaviour is not consistent when searching under the News & Events category.

**Comment by Tehsurfer** [comment]
@alan-wu ok waiting for their response on that ticket now

File: components/header/Header.vue
Comment (egauzens) [inline components/header/Header.vue:383]: typo? Should be search

**Review by egauzens — APPROVED** [review]
looks good to me once you fix that typo. Once it is merged in I will deploy it as a hotfix into prod along with a couple other changes that address the search for tools & resources

**Review by egauzens — COMMENTED** [review]
Sorry just realized the change you made for searching datasets. Why are we no longer going to the data page?

**Comment by egauzens** [comment]
Sorry again! I see what's happened, just confused myself alittle. Looks fine once you address the typo. Unfortunately Idk how to re-approve it once I dismissed it so you may need to request my review again or have alan approve it

**Pull request merged**
